frontend/src/Views/TransactionList/FilterSelect.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  Checkbox,
+  Input,
+  InputLabel,
+  ListItemText,
+  MenuItem,
+  Select
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+import React from "react";
+import { FilterItemArray } from "../../Models/TransactionFilterModel";
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
```

Comment:
neišieina šito padayt su `makeStyles`?

Reply:
Sita parsipuciau is mui docsu: https://material-ui.com/demos/selects/#multiple-select
Kitaip i ta menu neeina perduot props tik per sita, tai del to cia toks kinda clumsy sprendimas